### ISSUE TITLE
Use emoji-copy for emojis

### ISSUE THREAD

**Issue by benwaffle** [basecamp/omakub#251]
This is a super handy gnome extension - just press <kbd>Super</kbd> + <kbd>.</kbd>, type to search, and hit enter

https://github.com/FelipeFTN/Emoji-Copy

**Comment by justinhhorner** [comment]
Absolutely. I think we should have an emoji picker of some kind since this has been supported on macOS and Windows for some time.

Another approach we could take is to install the [Emoji extension](https://github.com/Ulauncher/ulauncher-emoji) for ULauncher. I've been using it for a few weeks now and it's worked very well.

**Comment by ardentperf** [comment]
FYI - i've had a bunch of issues with emojis.  First with fonts... I tried chrome at first and it was rendering all emojis with some font that had no color and was very poor; even after I installed fonts w apple emoji, twitter emoji and noto emoji. I never figured out the problem w chrome but realized that chromium didn't have the problem so i just switched. probably should have gone straight to brave instead of chromium, maybe will switch later but takes some time to move everything over.

second is the emoji picker. in gnome apps there's CTRL-. built in, however this doesn't work in chrome or chromium sadly. gnome has a built in "characters" app and i just assigned it to the shortcut key CTRL-ALT-SPACE (matching positions of CTRL-COMMAND-SPACE on mac). i'm not thrilled with the gnome emoji picker because the emojis are GIANT and the search isn't great.

i will definitely check out these extensions!

**Comment by dhh** [comment]
Going to stick with the emoji support we have for now through XCompose: https://manual.omakub.org/1/read/29/hotkeys#quick-emojis as the default.

**Comment by benwaffle** [comment]
That's fine, just wanted to say that the emoji-copy UX is much closer to macOS, and you don't need to memorize the hotkey combo for each emoji.
